src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputLiteralTypeConverter.cs

```diff
@@ -25,79 +25,56 @@ public override void Write(Utf8JsonWriter writer, InputLiteralType value, JsonSe
         public static InputLiteralType CreateInputLiteralType(ref Utf8JsonReader reader, string? id, string? name, JsonSerializerOptions options, ReferenceResolver resolver)
         {
             var isFirstProperty = id == null && name == null;
-            object? value = null;
-            InputType? type = null;
+            JsonElement? rawValue = null;
+            InputType? valueType = null;
             IReadOnlyList<InputDecoratorInfo>? decorators = null;
 
             while (reader.TokenType != JsonTokenType.EndObject)
             {
                 var isKnownProperty = reader.TryReadReferenceId(ref isFirstProperty, ref id)
-                    || reader.TryReadWithConverter(nameof(InputLiteralType.ValueType), options, ref type)
-                    || reader.TryReadWithConverter(nameof(InputLiteralType.Decorators), options, ref decorators);
+                    || reader.TryReadWithConverter("value", options, ref rawValue)
```

Comment:
which converter is this applied with?

Reply:
every type has a default converter if not specified.
This is invoking the default converter for `JsonElement`.
We could do this for all types.